.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Everything under the `.vscode` folder should not be added to source control. Please add that folder to the `.gitignore` file.